src/event.rs

```diff
@@ -281,6 +312,9 @@ pub struct Processor<N> {
     window_changes: WindowChanges,
     save_to_clipboard: bool,
     alt_send_esc: bool,
+    is_fullscreen: bool,
+    #[cfg(target_os = "macos")]
+    is_simple_fullscreen: bool,
```

Comment:
I've opened another PR for `winit` which allows you to query the `Window` itself to see if it's fullscreen or not: https://github.com/tomaka/winit/pull/838.

If adding these flags to `Processor` isn't preferred, we can wait for that PR to be merged, and then wait for `glutin` to be updated with the next version of `winit` and then use the `window.get_fullscreen()` methods.

Although to be honest, I think this way is fine as it is.